build-output-api/isr-revalidation/.vercel/output/functions/revalidate.func/index.js

```diff
@@ -0,0 +1,27 @@
+const fetch = require('node-fetch')
+
+// The bypass token can be a randomly generated string of at least 32 characters.
+// This is meant to be *private* - DO NOT expose this value on the client-side.
+const bypassToken = '87734ad8259d67c3c11747d3e4e112d0'
+
+const deployedUrl = 'https://isr-revalidation-endangeredmassa.vercel.app'
```

Comment:
I suppose we can construct this URL inside the HTTP handlers from `x-forwarded-proto` and `x-forwarded-host`?

Reply:
Yep, it should use `req.headers.host` instead

https://vercel.com/docs/concepts/edge-network/headers

Reply:
Updated!